client/js/views/templates/Home.js

```diff
@@ -8,30 +8,34 @@ module.exports = function( { model } ) {
 		vestibulum felis, ut sodales enim.</div>
 	<div>
 		<div>
-			<img class='pic' data-src='${ this.ImageSrc('Jam.jpg') }' alt='Jam'/>
-			<div class='text'>Nov. 12<br/>Wild Berry Jam</div>
+			<img data-src='${ this.ImageSrc('Jam.jpg') }' alt='Jam'/>
+	 		<div>Nov. 12<br/>Wild Berry Jam</div>
 		</div>
+		<div></div>
```

Comment:
This isn't needed.

Read about CSS flexbox and try to implement a solution using that.